docs/3.2. NMOS Private Claims.md

```diff
@@ -1,7 +1,9 @@
 # Private Claims
 _(c) AMWA 2021, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
 
-IS-10 has defined some private claims, related to the NMOS scopes. These claim names begin `x-nmos-` followed by the namespace identifier found in the URL for that given API (e.g. 'registration', 'query', 'connection', etc.). Each NMOS private claim specifies the read and write permission. Wildcards ('*') can be used to fine tune the level of access. In the following example, the token has been given full read and write permission for accessing the`registry.example.com` IS-04 Registration API.
+[IS-10 has defined some private claims](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.4._Behaviour_-_Access_Tokens.html#private-claims), related to NMOS systems. These claim names begin `x-nmos-` followed by the namespace identifier found in the URL for that given API (e.g. 'registration', 'query', 'connection', etc.). Each NMOS private claim specifies the read and write permission for that API. Wildcards ('\*') can be used to fine tune the level of access.
```

Comment:
I think that "scopes" may be confusing with that of the access token in the example below. So I am proposing this

```diff
-related to the NMOS scopes
+related to NMOS systems
```

Along with a link and fixing some formating 😄

Reply:
Agreed.  This section no longer exists in this doc, but I will try to remove reference to scopes where it is confusing.

Reply:
This was covered by #32